app/src/main/java/ch/epfl/sweng/runpharaa/notification/FireMessage.java

```diff
@@ -12,9 +10,6 @@
 import java.net.URL;
 
 public class FireMessage {
-    //Server key found in firebase
-    private final String SERVER_KEY = "AAAArozt-xs:APA91bEN2m2cVZVUlJ6GMRgacY42W7KF9f3Orpm1bcEKF0WbugfgDfwcKLZ1qmgs4d4EmD7iCgkB3qH6oXfjEk0vQEg2oodWioeYzve__QeCJr2PPgLI4qmHi71L5_EuxL87y1TJNlpX";
```

Comment:
I don't understand why you moved the two strings in the method...
And the final were needed as we do not want those strings to change 
(And as said in the previous PR about this, they should not appear in git so we need to correct this as well, maybe using git secret)

Reply:
My bad, AS asked to refactor this because the strings can be local variables in the only method they are called. I did not want to change this but I suspect I messed up one of my "revert to last commit". I restored this to how it was.